recipe/meta.yaml

```
@@ -19,7 +19,8 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - cmake
+    - {{ stdlib('c') }}
+    - cmake >=3.10
     - make   # [not win]
     - m2-make   # [win]
```

```suggestion
    - make
```

Wouldn't that be enough?

---

unfortunately didn't work on Windows :(